tests/test_load.py

```diff
@@ -224,7 +225,10 @@ def test_load_dataset_local(dataset_loading_script_dir, data_dir, keep_in_memory
             assert "Using the latest cached version of the module" in caplog.text
     with pytest.raises(FileNotFoundError) as exc_info:
         datasets.load_dataset("_dummy")
-    assert "at " + os.path.join("_dummy", "_dummy.py") in str(exc_info.value)
+    m_combined_path = re.search(fr"\S*{re.escape(os.path.join('_dummy', '_dummy.py'))}\b", str(exc_info.value))
+    assert m_combined_path is not None and os.path.isabs(m_combined_path.group())
+    m_path = re.search(r"\S*_dummy\b", str(exc_info.value))
+    assert m_path is not None and os.path.isabs(m_path.group())
```

Comment:
@lhoestq Actually, this check doesn't do anything (`m_path` returns a substring of `m_combined_path` without `.py` file extension). We can replace this check with a check which verifies that the error message returns a remote URL.
```python
m_paths = re.findall(r"\S*_dummy/_dummy.py\b", str(exc_info.value))  # on Linux this will match an URL as well as a local_path due to different os.sep, so take the last element (an URL always comes last in the list)
assert len(m_paths) > 0 and is_remote_url(m_paths[-1])
```